incubator/cockroachdb/values.yaml

```diff
@@ -0,0 +1,17 @@
+# Default values for cockroachdb.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+Name: "cockroachdb"
+Image: "cockroachdb/cockroach"
+ImageTag: "latest" # Keep at latest until 1.0, then switch to a specific version
```

Comment:
As noted until 1.0 it should stay as latest. Im fine with this especially since it is in the incubator.